packages/flutter_tools/lib/src/test/flutter_platform.dart

```diff
@@ -213,7 +213,7 @@ void catchIsolateErrors() {
 
 void main() {
   print('$_kStartTimeoutTimerMessage');
-  String serverPort = Platform.environment['SERVER_PORT'];
+  String serverPort = Platform.environment['SERVER_PORT'] ?? '';
```

Comment:
We can't yet declare this as `String?` or use '!', so just use `??` as we will fail either way.